CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## master
 
+- Require a minimum version of Ruby when installing the gem. ([@bibendi][])
```

Comment:
Please, add shortcut definition to the end of the file (currently, `@bibendi` link leads nowhere).